samples/src/main/java/com/mapbox/samples/test.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
how is this file used? Can't find any usages 🤔

Reply:
Whoops.